lib/internal/cluster/primary.js

```diff
@@ -85,9 +88,7 @@ cluster.setupPrimary = function(options) {
     return process.nextTick(setupSettingsNT, settings);
 
   initialized = true;
-  schedulingPolicy = cluster.schedulingPolicy;  // Freeze policy.
-  assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
-         `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
+  schedulingPolicy = SCHEDULE_POLICY_NAME_MAP.get(cluster.schedulingPolicy); // Freeze policy.
```

Comment:
Should we be keeping the `assert()`?

Reply:
As I understand, it should be handled via SafeMap:
https://github.com/yashLadha/node/blob/9027b55935784c5e40ac54db5919895ad1636f16/lib/internal/cluster/primary.js#L47-L49
so the value of `settings.schedulingPolicy` can be only supported in `SCHEDULE_POLICY_NAME_MAP`. Hence the `assert` is redundant in this particular case.